src/observable-membrane.ts

```diff
@@ -16,19 +17,17 @@ if (process.env.NODE_ENV !== 'production') {
     initDevFormatter();
 }
 
-export type ReactiveMembraneAccessCallback = (obj: any, key: ProxyPropertyKey) => void;
-export type ReactiveMembraneMutationCallback = (obj: any, key: ProxyPropertyKey) => void;
-export type ReactiveMembraneObservableCallback = (value: any) => boolean;
```

Comment:
Don't these have to be exported for TypeScript tooling to work correctly? Since they're used for `valueMutated`/`valueObserved`/`valueIsObservable` which are exposed.

Reply:
Those types were never exported outside this package because the `main.ts` only reexports the `ObservableMembrane`. This PR only removes the `export` as those types are not consumed anywhere else inside this package. 

This is also not an issue for typescript as non-exposed types are still preserved in the generated `.d.ts`.

We can always expose the types if the future if necessary.